packages/app-store/routing-forms/components/FormInputFields.tsx

```diff
@@ -2,7 +2,7 @@ import type { App_RoutingForms_Form } from "@prisma/client";
 import type { Dispatch, SetStateAction } from "react";
 
 import getFieldIdentifier from "../lib/getFieldIdentifier";
-import { getQueryBuilderConfig } from "../lib/getQueryBuilderConfig";
+import { getQueryBuilderConfigForFormFields } from "../lib/getQueryBuilderConfig";
```

Comment:
We have two different QueryBuilders now. One for FormFields that existed already and other one for Attributes that is new.